reflex/app.py

```diff
@@ -1535,3 +1564,12 @@ async def on_ping(self, sid):
         """
         # Emit the test event.
         await self.emit(str(constants.SocketEvent.PING), "pong", to=sid)
+
+    async def on_health(self, sid):
+        """Event for testing the Health of Reflex instance.
+
+        Args:
+            sid: The Socket.IO session id.
+        """
+        # Emit the test event.
+        await self.emit(str(constants.SocketEvent.HEALTH), "ok", to=sid)
```

Comment:
i don't think we need a websocket event handler as it's essentially equivalent to what we already have here for `ping`

Reply:
Yep, making the change